html/includes/api_functions.inc.php

```diff
@@ -1343,6 +1445,64 @@ function list_ipsec()
     api_success($ipsec, 'ipsec');
 }
 
+
+function list_vlans()
```

Comment:
Should also allow to pass hostname / device_id to limit the response.

Reply:
This will duplicate functionality of existing get_vlans function. I can add full listing into get_vlans, but i thought more simple functions better than fewer but with more nested conditions.

Reply:
Just realized that to have hostname listing is better in list functions, i will add it and in next pr will change get_vlans to more suitable form for get function, to get single vlan by its id.

Reply:
Thanks, forgot we had list_vlans but yes, better to just allow passing the hostname into that function and append an extra sql query.